.pylintrc

```diff
@@ -76,8 +77,10 @@ confidence=
 # --disable=W"
 
 disable=bare-except,
-        too-many-locals,
-        too-many-arguments
+	invalid-name,
```

Comment:
nit: indentation

Reply:
thanks for catching that. vim had inserted tabs for some reason